### ISSUE TITLE
[Bug] if rsnapshot_root contains a +, rm_rf will fail [sf#30]

### ISSUE THREAD

**Issue by bebehei** [rsnapshot/sourceforge-issues#42]
**Reported by craigregan on 2011-05-10 09:53 UTC**
If the snapshot_root directory contains a + in it, rsnapshot will fail with a message
rm_rf() tried to delete something outside of &lt;snapshot_root&gt; ... quitting now.

**Comment by bebehei** [comment]
I can't reproduce this.

```
bebe@rsnapshot:/tmp$ ./bin/rsnapshot -c etc/rsnapshot.conf.default alpha
bebe@rsnapshot:/tmp$ ./bin/rsnapshot --version
rsnapshot 1.3.1-145-g933f238
bebe@rsnapshot:/tmp$ grep -v '^#' etc/rsnapshot.conf.default | grep -v ^$
config_version  1.2
snapshot_root   /tmp/thisis+snapshots/
cmd_rm      /bin/rm
cmd_rsync   /usr/bin/rsync
retain  alpha   6
retain  beta    7
retain  gamma   4
verbose     2
loglevel    3
lockfile    /tmp/rsnapshot.pid
backup  /etc/       localhost/
```

**Comment by bebehei** [comment]
It also seems to be fixed with rsnapshot/rsnapshot@b6970c2f28c1af98664a4913e3740877dbb05ef8
